installation.md

```diff
@@ -76,6 +76,27 @@ Or to have convenience commands for:
 The most recent version of the tool can be retrieved from the [official release
 page](https://github.com/kubevirt/kubevirt/releases).
 
+## Deploying on OpenShift
+A couple of privileges must be granted prior `kubevirt.yaml` deployment.
+To do so, the `kubevirt` service account is required:
+
+```
+oc create serviceaccount -n kube-system kubevirt
+```
+
+and following [SCCs](https://docs.openshift.com/container-platform/3.7/admin_guide/manage_scc.html) needs to be added:
+```
+oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:kubevirt-privileged
+oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:kubevirt-controller
+```
+
+Please note, the `kubevirt` service account is created even within `kubevirt.yaml` but creating it explicitly allows adding SCCs before `kubevirt.yaml` is deployed.
```

Comment:
How does this make a difference?

Reply:
oh, right. No difference, declarative approach.